### ISSUE TITLE
Default visual focus indicator is low contrast on white backgrounds

### ISSUE THREAD

**Issue by cmajel** [cal-itp/benefits#3265]
After seeing a low contrast focus item here in #3264, it appears the default visual focus color, #fdb714/ var(--focus-color), doesn't meet a [3:1 contrast ratio on white backgrounds per WCAG.](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html) 

This issue would evaluate the existing Cal-ITP Benefits color space and suggest an alternative, ideally one focus color that would work on both light and dark backgrounds.

[Figma file with initial exploration ](https://www.figma.com/design/SeSd3LaLd6WkbEYhmtKpO3/Benefits--Full-Application-?node-id=21821-6679&t=NfD3dFFTQ5TG1nib-0)

---

## Screenshots

<img width="844" height="841" alt="Image" src="https://github.com/user-attachments/assets/b6a00362-770b-45eb-97ad-c8de0d9a5796" />

<img width="794" height="913" alt="Image" src="https://github.com/user-attachments/assets/a1a11c3a-1183-4b78-a5f6-951225e90440" />

**Comment by thekaveman** [comment]
@cmajel it looks good to me, thank you for this effort!

~Let's keep this ticket open, and we'll get a PR up to fix the color / spacing and close it out with that~

Sorry, I forgot we decided not to do this, so Design can close out its efforts separately from any dev work. I think this is good to merge the Figma branch and close!

**Comment by thekaveman** [comment]
Thank you @cmajel 

Just to be extra sure, here are the ones I was confused about, and what I understand now:

* Language button in the "navbar" (dark blue background): `#FEBD28`
* Both the drop-down and button inputs on the homepage (dark / black background): `#FEBD28`
* Links in the footer (dark / black background): `#FEBD28`

And for all others (white background): `#756FFF`
